nengo/spa/state.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+
+import nengo
+from nengo.spa.module import Module
+from nengo.networks.ensemblearray import EnsembleArray
+
+
+class State(Module):
+    """A module capable of representing a single vector, with optional memory.
+
+    This is a minimal SPA module, useful for passing data along (for example,
+    visual input).
+
+    Parameters
+    ----------
+    dimensions : int
+        Number of dimensions for the vector
+    subdimensions : int
+        Size of the individual ensembles making up the vector. Must divide
+        evenly into dimensions
+    neurons_per_dimensions : int
+        Number of neurons in an ensemble will be this*subdimensions
+    float : float
+        Gain of feedback connection. Set to 1.0 for perfect memory. Other
+        non-zero values will create a decaying memory
+    feedback_synapse : float, nengo.Synapse
+        The synapse on the feedback connection
+    vocab : Vocabulary, optional
+        The vocabulary to use to interpret this vector
+    """
+
+    def __init__(self, dimensions, subdimensions=16, neurons_per_dimension=50,
+                 feedback=0.0, feedback_synapse=0.01, vocab=None, label=None,
```

Comment:
The previous discussion (that I'd still agree with) was that the ```tau``` parameter was a bad idea.

As for changing the default, yes, i'd agree with @xchoo that a better default would be 0.1.  I find myself changing it to that almost all the time....